include/geos/geom/Location.h

```diff
@@ -30,7 +30,8 @@ namespace geom { // geos::geom
  * For a description of the DE-9IM, see the
  * [OpenGIS Simple Features Specification for SQL](http://www.opengis.org/techno/specs.htm").
  */
-GEOS_DLL enum class Location : char {
+#undef Location
```

Comment:
What is this `undef` needed for?

Reply:
killed debugging lint in 2387764a226199beb7d5b9bdc5c2cf81960b4f1c